openapi/data_repository_service.swagger.yaml

```diff
@@ -194,57 +194,71 @@ definitions:
           sha512
   Bundle:
     type: object
-    required: ['id', 'object_ids', 'created', 'updated', 'version', 'checksums']
     properties:
       id:
         type: string
-        description: |-
+        description: >-
           An identifier, unique to this Data Bundle
       object_ids:
```

Comment:
isn't `object_ids` redundant, now that we have `contents` ?

Reply:
Good catch. I forgot to remove it. Removed in 03c1b01ba49227f8f15737b8a282876e40946f58